src/partition/repart.c

```diff
@@ -103,6 +103,14 @@ static enum {
         EMPTY_CREATE,   /* create disk as loopback file, create a partition table always */
 } arg_empty = EMPTY_REFUSE;
 
+typedef enum {
```

Comment:
not that it matters, but usually we give all enums a name, and the typedef too. i.e. we'd use `typedef enum Foo { … } Foo`